binding.gyp

```diff
@@ -6,7 +6,7 @@
        'src/bindings.cc'
     ],
     'include_dirs' : [
-      "<!@(node -p \"require('node-addon-api').include\")"
+      "<!@(node -p \"require('node-addon-api').include_dir\")"
```

Comment:
```suggestion
      "<!(node -p \"require('node-addon-api').include_dir\")"
```

Reply:
Thanks for the suggestion, I don't have access to Windows box to fix the build failure.